code/modules/mob/living/simple_animal/hostile/megafauna/seer.dm

```diff
@@ -0,0 +1,434 @@
+/mob/living/simple_animal/hostile/megafauna/seer/proc/howling_charge()
+	var/turf/T = get_turf(target)
+	if(QDELETED(T) && T == get_turf(src))
+		return
+	new/obj/effect/temp_visual/dragon_swoop/bubblegum(T)
+	charging = TRUE
+	setDir(get_dir(src, T))
+	visible_message("<span class='revenboldnotice'>[src] howls and prepares to charge!</span>")
+	playsound(src, 'sound/effects/curseattack.ogg', 75, TRUE, frequency = 0.9)
+	sleep(3)
+	throw_at(T, get_dist(src, T), 1, src, 0)
+
+/mob/living/simple_animal/hostile/megafauna/seer/throw_impact(atom/A)
+	if(form != FORM_BUBBLEGUM || !charging || !isliving(A))
+		. = ..()
+	else
+		var/mob/living/L = A
+		L.visible_message("<span class='revenboldnotice italics'>[src] slams into [L]!</span>", \
+		"<span class='userdanger'>[src]'s shadowy body passes through you, and churning darkness envelops your vision!</span>")
+		L.apply_damage(5, BURN)
+		L.apply_necropolis_curse(CURSE_BLINDING)
+		playsound(L, 'sound/effects/meteorimpact.ogg', 100, TRUE)
+		shake_camera(src, 2, 3)
+		shake_camera(L, 4, 3)
+		var/throwtarget = get_edge_target_turf(src, get_dir(src, get_step_away(L, src)))
+		L.throw_at(throwtarget, 3)
+	charging = FALSE
+
+
+
+//Vacuous Spiral: Colossus form attack. Emits a spiral of bolts.
+/mob/living/simple_animal/hostile/megafauna/seer/proc/vacuous_spiral()
+	set waitfor = FALSE
+	var/counter = 1
+	var/turf/marker
+	for(var/i in 1 to 40)
+		switch(counter)
+			if(1)
+				marker = locate(x, y - 2, z)
+			if(2)
+				marker = locate(x - 1, y - 2, z)
+			if(3)
+				marker = locate(x - 2, y - 2, z)
+			if(4)
+				marker = locate(x - 2, y - 1, z)
+			if(5)
+				marker = locate(x - 2, y, z)
+			if(6)
+				marker = locate(x - 2, y + 1, z)
+			if(7)
+				marker = locate(x - 2, y + 2, z)
+			if(8)
+				marker = locate(x - 1, y + 2, z)
+			if(9)
+				marker = locate(x, y + 2, z)
+			if(10)
+				marker = locate(x + 1, y + 2, z)
+			if(11)
+				marker = locate(x + 2, y + 2, z)
+			if(12)
+				marker = locate(x + 2, y + 1, z)
+			if(13)
+				marker = locate(x + 2, y, z)
+			if(14)
+				marker = locate(x + 2, y - 1, z)
+			if(15)
+				marker = locate(x + 2, y - 2, z)
+			if(16)
+				marker = locate(x + 1, y - 2, z)
+		counter++
+		if(counter > 16)
+			counter = 1
+		if(counter < 1)
+			counter = 16
+		vacuous_bolt(marker)
+		playsound(src, 'sound/magic/clockwork/invoke_general.ogg', 10, TRUE)
+		playsound(src, "curse", 30, TRUE)
+		sleep(1)
+
+/mob/living/simple_animal/hostile/megafauna/seer/proc/vacuous_bolt(turf/marker)
+	if(!marker || marker == loc)
+		return
+	var/turf/startloc = get_turf(src)
+	var/obj/item/projectile/P = new /obj/item/projectile/vacuous_bolt(startloc)
+	P.current = startloc
+	P.starting = startloc
+	P.firer = src
+	P.yo = marker.y - startloc.y
+	P.xo = marker.x - startloc.x
+	if(target)
+		P.original = target
+	else
+		P.original = marker
+	P.fire()
+
+/obj/item/projectile/vacuous_bolt
+	name = "vacuous bolt"
+	icon_state = "greyscale_bolt"
```

Comment:
I'll see if I can sneak it in somehow, but I was trying to make it a "negative energy"-esque version of the normal death bolt. It looks mostly the same, but _spookier._